src/components/text_link.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react'
+import styled from 'styled-components'
+import colors from '../assets/colors'
+import '../assets/fonts'
+
+
+export interface LinkProps extends React.Props<TextLink>, React.HTMLAttributes<TextLink> {
+  href?: string
+  underline?: boolean
+}
+
+export class TextLink extends React.Component<LinkProps, null> {
+  public static defaultProps: LinkProps
+
+  render() {
+    return (
+      <a href={this.props.href} className={this.props.className}>
+        {this.props.children}
+      </a>
+    )
+  }
+}
+
+TextLink.defaultProps = {
+    underline: false
```

Comment:
I'm guessing there's a couple uses of this you do want the underline for? Like the 'Enter Live Auction', or 'Contact Gallery'? Not sure, but I guess the 'styling' can be done later.